app/src/main/java/org/sopt/and/presentation/home/HomeViewModel.kt

```diff
@@ -0,0 +1,38 @@
+package org.sopt.and.presentation.home
+
+import androidx.annotation.StringRes
+import androidx.lifecycle.ViewModel
+import kotlinx.coroutines.flow.MutableStateFlow
+import kotlinx.coroutines.flow.asStateFlow
+import org.sopt.and.R
+
+
+class HomeViewModel : ViewModel() {
+    private val _uiState = MutableStateFlow(HomeUiState())
+    val uiState = _uiState.asStateFlow()
+}
+
+data class HomeUiState(
+    val bannerContent: List<ContentItem> = listOf(
+        ContentItem(R.string.banner_example_1, R.drawable.ic_launcher_background),
+        ContentItem(R.string.banner_example_2, R.drawable.ic_launcher_background),
+        ContentItem(R.string.banner_example_3, R.drawable.ic_launcher_background)
+    ),
+    val editorPicks: List<ContentItem> = listOf(
+        ContentItem(R.string.editor_example_1, R.drawable.ic_launcher_background),
+        ContentItem(R.string.editor_example_2, R.drawable.ic_launcher_background),
+        ContentItem(R.string.editor_example_3, R.drawable.ic_launcher_background)
+    ),
+    val top20: List<ContentItem> = listOf(
+        ContentItem(R.string.top_example_1, R.drawable.ic_launcher_background),
+        ContentItem(R.string.top_example_2, R.drawable.ic_launcher_background),
+        ContentItem(R.string.top_example_3, R.drawable.ic_launcher_background),
+        ContentItem(R.string.top_example_3, R.drawable.ic_launcher_background)
+    )
+)
+
+
+data class ContentItem(
+    @StringRes val title: Int,
+    val imageRes: Int
```

Comment:
```suggestion
    @DrawableRes val imageRes: Int
```
이렇게 어노테이션으로 Drawable을 표시하는 거라고 명시해주면 좋을 것 같아요!